iBudget/spending/views.py

```diff
@@ -55,8 +57,34 @@ def show_spending_group(request):
 
 
 @require_http_methods(["POST"])
-def set_spending_limitation_ind(request):
-    """Handling request for create spending limitation.
+def set_limitation_period(request):
+    """Handling request for defining limitation period.
+
+        Args:
+            request (HttpRequest): Type of period.
+        Returns:
+            HttpResponse object.
+    """
+    user = request.user
+    data = json.loads(request.body)
+    if data['fixed'] == 'null':
+        return HttpResponse('Bad request', status=400)
+    data = data['fixed']
+    user.ind_period_fixed = data
+    print(data)
```

Comment:
It is better to use logger instead of prints.

Reply:
It's my fault, I'll delete it. Thank you